apps/web/src/lang/fr.json

```diff
@@ -7359,7 +7347,7 @@
     "description": "Error message when pool candidates could not be loaded"
   },
   "ZWnKEJ": {
-    "defaultMessage": "Famille de compétences {skillFamilyId} non trouvée.",
+    "defaultMessage": "Groupe de compétences {skillFamilyId} non trouvée.",
```

Comment:
This should say "Groupe de compétences {skillFamilyId} non trouvé."

Reply:
## French screenshot
<img width="1440" alt="Screen Shot 2024-11-29 at 15 29 16" src="https://github.com/user-attachments/assets/20191696-16b2-4ebf-be0f-6aec8edc8164">


I noticed the English string needed to be adjusted as well.

## English screenshot before
<img width="592" alt="Screen Shot 2024-11-29 at 15 33 11" src="https://github.com/user-attachments/assets/cd186357-b689-4bf3-b1d4-b018ec245d23">

## English screenshot after
<img width="558" alt="Screen Shot 2024-11-29 at 15 33 23" src="https://github.com/user-attachments/assets/a256ea52-4dfb-42fb-9beb-4135dbbb7ce8">